src/foundry/common/primitives/string.mjs.d.ts

```diff
@@ -0,0 +1,39 @@
+interface String {
+  /**
+   * Capitalize a string, transforming it's first character to a capital letter
+   */
+  capitalize<S extends string>(this: S): Capitalize<S>;
```

Comment:
There are some tests for this in a `primitives.mjs.test-d.ts` file. They should be moved to a `string.mjs.test-d.ts` file.